src/libs/actions/Report.js

```diff
@@ -875,6 +875,21 @@ function handleReportChanged(report) {
     // Store the max sequence number for each report
     reportMaxSequenceNumbers[report.reportID] = report.maxSequenceNumber;
 
+    // Store the max sequence number for each report excluding loading actions
+    Onyx.connect({
+        key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${report.reportID}`,
+        callback: actions => reportMaxSequenceNumbersNotLoading[report.reportID] = _.chain(actions)
+
+            // We want to avoid marking any pending actions as read since
+            // 1. Any action ID that hasn't been delivered by the server is a temporary action ID.
+            // 2. We already set a comment someone has authored as the lastReadActionID_<accountID> rNVP on the server
+            // and should sync it locally when we handle it via Pusher or Airship
+            .reject(action => action.loading)
+            .pluck('sequenceNumber')
+            .max()
+            .value(),
+    });
+
```

Comment:
@marcaaron I'm moving this logic here as requested, but it does require an ONYX call